MultiVersions/Forge/main/loginar/lang/zh_tw.json

```
@@ -0,0 +1,36 @@
+{
+  "block.loginar.huge_loginar_urn": "特大型洛琪納物品甕",
+  "block.loginar.large_loginar_urn": "大型洛琪納物品甕",
+  "block.loginar.medium_loginar_urn": "中型洛琪納物品甕",
+  "block.loginar.small_loginar_urn": "小型洛琪納物品甕",
+  "block.loginar.super_loginar_urn": "巨型洛琪納物品甕",
+  "block.loginar.tiny_loginar_urn": "迷你洛琪納物品甕",
+  "container.loginar.loginar_urn": "洛琪納物品甕",
+  "container.loginar.lunch_box": "午餐盒",
+  "entity.loginar.loginar": "洛琪納",
+  "item.loginar.backpack_upgrade": "背包化升級",
+  "item.loginar.backpack_upgrade.desc": "讓甕不用放置也能打開",
```

為了翻譯一致性，`Open` 習慣上是譯作 `開啟`
```suggestion
  "item.loginar.backpack_upgrade.desc": "讓甕無須放置，能直接在手上開啟",
```